src/Attribute/Parameter/CastToString.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Hydrator\Attribute\Parameter;
+
+use Attribute;
+use Stringable;
+use Yiisoft\Hydrator\Context;
+use Yiisoft\Hydrator\NotResolvedException;
+use Yiisoft\Hydrator\ParameterAttributeInterface;
+use Yiisoft\Hydrator\ParameterAttributeResolverInterface;
+use Yiisoft\Hydrator\UnexpectedAttributeException;
+
+use function is_bool;
+use function is_float;
+use function is_int;
+use function is_null;
+use function is_string;
+
+#[Attribute(Attribute::TARGET_PROPERTY | Attribute::TARGET_PARAMETER | Attribute::IS_REPEATABLE)]
```

Comment:
Theoretically, yes.

Value pass to chain attributes A1 → A2 → A3... May be in something edge case it will be necessary to use `CastToString` several times.

Reply:
Yes. In separate PR.